libraries/ESP8266MQTTClient/src/URIParser.h

```diff
@@ -0,0 +1,28 @@
+#ifndef URIParser_h_
```

Comment:
I think URIParser can be it's own thing and be used by other libs as well :) it's kinda great